changelogs/unreleased/5543-resource-file-upload-should-ignore-identical-files.yml

```diff
@@ -0,0 +1,8 @@
+description: The stat_file API endpoint now returns a list with no duplicates.
+change-type: patch
+destination-branches:
+  - master
+  - iso6
+  - iso5
+sections:
+  bugfix: "{{description}}"
```

Comment:
I don't have a strong opinion about this changelog entry being present or not

Reply:
I think the message is good as an internal description of the change. For the end-user message I'd focus more on the symptom: this fixes a bug that may cause the first export for an environment to fail if there are identical files.

Reply:
Great point :+1: